package.json

```diff
@@ -29,16 +32,24 @@
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.18.6",
     "@docusaurus/module-type-aliases": "2.2.0",
+    "@linaria/webpack-loader": "^4.1.9",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^12.1.5",
     "@testing-library/user-event": "^14.4.3",
     "@tsconfig/docusaurus": "^1.0.5",
     "babel-jest": "^29.3.1",
+    "babel-loader": "^9.1.0",
```

Comment:
please remove `babel-loader`, `css-loader`, `file-loader`, `mini-css-extract-plugin`, `webpack`, `webpack-cli`, `webpack-dev-server`, we don't need them, thanks thanks :pray: